__mocks__/dh-core.js

```diff
@@ -466,17 +466,22 @@ class Format {
 }
 
 class FilterCondition {
+  constructor(type, parent) {
```

Comment:
Doesn't look like you ended up using these, you just defined a new mock in `TableUtils.test.ts`. Revert the changes to this file.